scripts/secure-boot/generate-images/generate-image.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+SCRIPTDIR=$(readlink -f $(dirname "$0"))
+COUNT=$1
+TESTNAME=$2
+N=$3
+
+# go into a separate directory to make cleanup easier
+TEMPDIR=$(mktemp -d)
+cd "$TEMPDIR"
```

Comment:
```
In scripts/secure-boot/generate-images/generate-image.sh line 11:
cd "$TEMPDIR"
^-----------^ SC2164 (warning): Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```

You could also use set -e as mentioned earlier.

Reply:
we now use [set -e](https://github.com/Dasharo/open-source-firmware-validation/pull/192/commits/bce10d72eb89079d32fb6db71846782fa2f3e593#diff-00f53487a7d91b72da783fcf7286011bd2c964cb60496c9c6e607a55e9ae748dR3), in both scripts